### PR TITLE
virttest/utils_test/qemu.py: Add random time before migration

### PR DESCRIPTION
**Pull request by spcui**
Sometimes, we need wait random time before migration.

Signed-off-by: Shuping Cui <scui@redhat.com>

ID: 1256109

### CODE REVIEW

File: virttest/utils_test/qemu.py [hunk 1]
```diff
@@ -813,6 +814,8 @@ def migrate_wrap(vms_name, srchost, dsthost, start_work=None,
                         for vm in mig_data.vms:
                             vm.resume()
                         wait = self.params.get("start_migration_timeout", 0)
+                        if wait == "random":
```
Comment (CongSmile) [inline virttest/utils_test/qemu.py]: Why not if self.params.get() == "***":  ?

**Comment by CongSmile** [comment]
Acked.

**Comment by PandaWei** [comment]
sometimes?
which case we need to wait *random* time?
No 'random' is found in cfg file.
can you explain it clearly？

**Comment by spcui** [comment]
Hi @PandaWei 
https://github.com/autotest/tp-qemu/pull/450/files this one

**Comment by vipmike007** [comment]
Acked-By: Mike Cao <bcao@redhat.com>

**Comment by PandaWei** [comment]
@spcui 
Thanks for your response firstly.
But I doubt the rationality of the range from 5 to 20.
And It's bad idea that you set the range value to be fixed value in API function.


But i think it's a bad idea add random time here.
avocado-vt,

**Comment by PandaWei** [comment]
Maybe you can make range as a variable in cfg file
- timeout_random:
  start_migration_timeout = random
  min_boundary = XXX
  max_boundary = YYY

**Comment by spcui** [comment]
Hi @PandaWei 
OK, thanks your comments, will update later.

**Comment by spcui** [comment]
Close it, will set random time in config file.

**Pull request closed without merging**